src/main.cpp

```diff
@@ -143,10 +143,17 @@ struct JsonFileFinder
             return;
         }
 
+        // convert windows new-lines to UNIX by removing
+        // all instances of '\r'
+        std::string markerCheckStr = str;
+        std::string::iterator it = std::remove(markerCheckStr.begin(), markerCheckStr.end(), '\r');
+        markerCheckStr.erase(it, markerCheckStr.end());
+
         // there might be non-clang time trace json files around;
         // the clang ones should have this inside them
         const char* clangMarker = "{\"cat\":\"\",\"pid\":1,\"tid\":0,\"ts\":0,\"ph\":\"M\",\"name\":\"process_name\",\"args\":{\"name\":\"clang\"}}";
-        if (strstr(str.c_str(), clangMarker) == NULL)
+        const char* newClangMarker = "{\n      \"args\": {\n        \"name\": \"clang\"\n      },\n      \"cat\": \"\",\n      \"name\": \"process_name\",\n      \"ph\": \"M\",\n      \"pid\": 1,\n      \"tid\": 0,\n      \"ts\": 0\n    }";
+        if (strstr(markerCheckStr.c_str(), clangMarker) == NULL && strstr(markerCheckStr.c_str(), newClangMarker) == NULL)
             return;
```

Comment:
Hmm maybe this should get a bit relaxed treatment, like (typing out of the top of my head, might not compile):

```
const char* kClangMarker1 = "\"name\":\"clang\"";
const char* kClangMarker2 = "\"name\": \"clang\"";
const char* kProcessNameMarker1 = "\"name\":\"process_name\"";
const char* kProcessNameMarker2 = "\"name\": \"process_name\"";
if (strstr(str.c_str(), kClangMarker1) == NULL && strstr(str.c_str(), kClangMarker2) == NULL)
    return;
if (strstr(str.c_str(), kProcessNameMarker1) == NULL && strstr(str.c_str(), kProcessNameMarker2) == NULL)
    return;
```
and then there's no need to do newline conversion.